### PR TITLE
📈 Add cliVer parameter to log event functions and components

### PR DESCRIPTION
**Pull request by FunamaYukina**
## Summary

Add ``cliVer`` parameter to log event functions and components.

ss 2573

### CODE REVIEW

**Review by MH4GF — APPROVED** [review]
I'm thinking it could be common, but it looks good because it's a temporary solution. 👍🏻 
Thanks!!

File: frontend/.changeset/eighty-bottles-hope.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@liam-hq/erd-core": patch
+"@liam-hq/cli": patch
+---
+
+📈 Add cliVer parameter to log event functions and component.s
```
Comment (hoshinotsuyoshi) [inline frontend/.changeset/eighty-bottles-hope.md:6]: nits

```suggestion
📈 Add cliVer parameter to log event functions and components
```

Comment (FunamaYukina) [reply frontend/.changeset/eighty-bottles-hope.md:6]: Oh, sorry🤦‍♀️. I made mistake...

Comment (MH4GF) [reply frontend/.changeset/eighty-bottles-hope.md:6]: oh!!
I wonder if I can rewrite the changeset later. I'll give it a try.

**Pull request merged**